x-pack/legacy/plugins/siem/public/components/filters_global/filters_global.tsx

```diff
@@ -17,7 +17,8 @@ const offsetChrome = 49;
 const gutterTimeline = '70px'; // Temporary until timeline is moved - MichaelMarcialis
 
 const disableSticky = 'screen and (max-width: ' + euiLightVars.euiBreakpoints.s + ')';
-const disableStickyMq = window.matchMedia(disableSticky);
+const disableStickyMq =
+  typeof window.matchMedia === 'function' ? window.matchMedia(disableSticky) : { matches: false };
```

Comment:
Although we've mocked window.matchMedia here,
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/public/mock/match_media.ts
but somehow still complaining about `window.matchMedia is not a function` when I ran the test.
Therefore checking the type for window.matchMedia here to get rid of the error.
Would like to know if anyone met this before or if I missed anything?